test/integration/provider.spec.ts

```diff
@@ -1,10 +1,26 @@
 import { CHAIN_ID, NetworkName } from '../../src'
-import { provider } from './setup'
+import { provider, grpcProvider } from './setup'
 import { strToBytes, U256 } from '../../src/basicElements/serializers'
+import { PublicStatus } from '../../src/generated/grpc/massa/model/v1/node_pb'
 
 const USDC = 'AS12k8viVmqPtRuXzCm6rKXjLgpQWqbuMjc37YHhB452KSUUb9FgL'
 
-describe('Provider tests', () => {
+describe('Provider GRPC tests', () => {
+  test('getNodeStatus', async () => {
+    const status = (await grpcProvider.getNodeStatus()) as PublicStatus
+    expect(status.getConfig()).toBeDefined()
+    expect(status.getCurrentCycle()).toBeDefined()
+    expect(status.getCurrentTime()).toBeDefined()
+    expect(status.getCurrentCycleTime()).toBeDefined()
+    expect(status.getLastExecutedFinalSlot()).toBeDefined()
+    expect(status.getLastExecutedSpeculativeSlot()).toBeDefined()
+    expect(status.getFinalStateFingerprint()).toBeDefined()
+    expect(status.getChainId()).toBeDefined()
+    expect(status.getMinimalFees()).toBeDefined()
+  })
+})
```

Comment:
It would be nice to have more tests. I suggest to put them in a dedicated file. this can be done in a followup issue

Reply:
done